Automattic/wp-calypso
# Stats: Update slider thumb to match design

## Proposed Changes

Draws SVG "<>" in thumb instead of noting the current step value.

<img width="423" alt="SCR-20231207-pzje" src="https://github.com/Automattic/wp-calypso/assets/40267301/c04a4485-56da-4f01-8d99-6eaaa682574f">

## Testing Instructions

* Open Calypso Live Branch
* Open purchase page, and apply feature flag `stats/tier-upgrade-slider`
* Ensure the thumb looks like the design

## Pre-merge Checklist
